.github/workflows/extract-translation-source-files.yml

```diff
@@ -214,7 +245,7 @@ jobs:
           cd translations/${{ matrix.repo }}
           # finds the directory containing the english locale usually located in
           # */*/conf/locale/en
-          EN_DIR=$(find . -type d -regex ".+conf\/locale\/en$")
+          EN_DIR=$(find . -type d -regex ".+conf\/locale\/en$" | head -n 1)
```

Comment:
@OmarIthawi , setting it as draft so I can fix it tomorrow morning. This addition `| head -n 1` is causing a problem with other repos like `ecommerce`

Reply:
Please review now @OmarIthawi . It works fine :) . The script works for `edx-platform` and other repos. Tested in fork. Also notice that we don't delete contents of `conf/locale` anymore